src/lib/accounts/accounts.test.ts

```diff
@@ -93,4 +101,32 @@ describe('lib/accounts test suite', () => {
     const ok = await ac.createAgreement('my-doc', '7fab36d8-a63a-4543-9a24-d7a3fe2f128b');
     expect(ok).toBeTruthy();
   });
+
+  it('should return null when a user cannot be found', async () => {
+    const ac = new AccountsClient(cfg);
+    const user = await ac.getUser('4f11eb3b-f45c-4fd3-9241-533d29a0582b');
+    expect(user).toBeNull();
+  });
+
+  it('should pass along any errors from a non-404', async () => {
+    const ac = new AccountsClient(cfg);
+    await expect(ac.getUser('1d2a9ece-3d06-4aa7-bffc-c521cf7ef6cb')).rejects.toThrowError();
+  });
+
+  it('should get a user', async () => {
+    const ac = new AccountsClient(cfg);
+    const user = await ac.getUser('7fab36d8-a63a-4543-9a24-d7a3fe2f128b');
+    expect(user).toBeTruthy();
+
+    if (user) {
+      expect(user.uuid).toEqual('7fab36d8-a63a-4543-9a24-d7a3fe2f128b');
```

Comment:
I'm wondering whether the `if (user) {` guard is necessary but I don't know this testing library well. Will `expect(user).toBeTruthy();` exit this function immediately if it fails?

Reply:
@AP-Hunt tells me this was autoformatted to comply with `tslint`. Resolving…